client/core/core.go

```diff
@@ -1207,6 +1210,28 @@ func (c *Core) handleUnbookOrderMsg(dc *dexConnection, msg *msgjson.Message) err
 	return ob.Unbook(&note)
 }
 
+// handleEpochOrderMsg is called when an epoch_order notification is
+// received.
+func (c *Core) handleEpochOrderMsg(dc *dexConnection, msg *msgjson.Message) error {
+	var note msgjson.EpochOrderNote
+	err := json.Unmarshal(msg.Payload, &note)
+	if err != nil {
+		return fmt.Errorf("epoch order note unmarshal error: %v", err)
+	}
+
+	dc.epochsMtx.Lock()
+	eq, ok := dc.epochs[note.MarketID]
```

Comment:
I suspect we can/should ignore epoch notes for markets Core isn't using according to Core.Markets